spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1189,6 +1189,19 @@
                 :idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review,
               )
             end
+
+            it 'does not deactivate the profile when the enrollment has expired' do
```

Comment:
For both tests- the profile at this point is not active so we don't deactivate it, correct? Maybe we can reword each tests around what is happening with fraud rejection?